### DISCUSSION TITLE
0.9.6 Snapshot

### DISCUSSION THREAD

**Discussion by mariuz** [mariuz/flamerobin#293] [General]
## What's Changed
* Information about the error in the LoadLibrary function by @Jdochoa in https://github.com/mariuz/flamerobin/pull/291

**Full Changelog**: https://github.com/mariuz/flamerobin/compare/0.9.5...0.9.6

<hr /><em>This discussion was created from the release <a href='https://github.com/mariuz/flamerobin/releases/tag/0.9.6'>0.9.6 Snapshot</a>.</em>

**Comment by mrjohnr** [comment]
seems that setup doesn't install xml-styles folder, I got is from sources zip

**Comment by ron90** [comment]
After installing from the setup the following error comes up in flamerobin  

![Error2](https://user-images.githubusercontent.com/65551754/208354184-81daa0d1-6115-4b4e-9711-d419ff2858d0.png)

I also found that xml-styles folder is missing in the installation folder. After copying it from the source, I also had to assign it write/modify permission at OS level to allow flamerobin to save changes to it.

The subsequent error
![Error1](https://user-images.githubusercontent.com/65551754/208354835-b811243a-6be5-434b-baf2-b3670c17b0c7.png)
can be solved by saving a theme in preferences as mentioned in issue #292

**Comment by Jdochoa** [comment]
@mariuz  What script do you use for the installer?

**Comment by arvanus** [comment]
I hadn't time to do, but I'll fix the installer soon

**Comment by arvanus** [comment]
Please, download and install the latest release file again
Should be fixed now

**Reply by fperana** [reply]
I've installed from the latest available installers and got the same errors, plus another two:
1) Flamerobin 32bit tries to load fbclient.dll from my local firebird installation, which is 64bit, so it fails to load (Flamerobin 0.9.3.1bd8c1ac didn't have this problem)
2) Opening the DDL of a Stored Procedure in the editor caused a fatal error, which in turn caused Flamerobin to crash and close.
